### PR TITLE
generate enter room poker token

### PR DESCRIPTION
**Pull request by naijgnay**

### CODE REVIEW

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/config/interceptors/CheckRoomPasswordIntercepter.java [hunk 1]
```diff
@@ -19,13 +22,22 @@
...
+        if (roomToken != null && roomToken.length == 1) {
+            Claims claims = JwtUtil.parseJWT(roomToken[0]);
+            hasToken = claims != null && "roomPassword".equals(claims.getSubject());
+        }
         String roomName = request.getRequestURI().substring(request.getRequestURI().lastIndexOf("/") + 1);
         String cookieKey = "roomPassword_".concat(roomName);
         String roomPassword = CookieUtil.extractCookie(request, cookieKey);
-        Optional<Room> findRoom = webAgentSessionRegistry.getRooms().stream().filter(roomItem -> (roomItem.getName().equals(roomName) && (roomItem.getRoomPassword().equals(roomPassword)))).findAny();
-        if(findRoom.isPresent()) {
+        Optional<Room> findRoom = webAgentSessionRegistry.getRooms().stream().filter(roomItem -> (roomItem.getName().equals(roomName) && (roomItem.getRoomPassword().equals(roomPassword) ))).findAny();
```
Comment (TRUEJASONFANS) [inline sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/config/interceptors/CheckRoomPasswordIntercepter.java]: why not add a method in Room?

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/util/JwtUtil.java [hunk 2]
```diff
@@ -13,11 +13,11 @@
  * Created by jianyang on 2019/1/9.
  */
 
-@ConfigurationProperties("jwt.config")
+//@ConfigurationProperties("jwt.config")
 @Component
 public class JwtUtil {
-    private static String  key; //密钥
-    private static long ttl;   //过期时间
+    private static String  key = "1234"; //密钥
```
Comment (TRUEJASONFANS) [inline sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/util/JwtUtil.java]: too simple key

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java [hunk 3]
```diff
@@ -49,14 +56,29 @@ public ResponseResult checkRoomPassword(@RequestBody Room room, HttpServletRespo
...
+        passwordCookie.setPath("/pockerRoom");
         response.addCookie(passwordCookie);
         return new ResponseResult(ResponseResult.SUCCESS, "房间密码正确");
       } else {
         return new ResponseResult(ResponseResult.FAIL, "房间密码不正确！");
       }
   }
 
+  @ResponseBody
+  @PostMapping("/api/roomToken/{roomName}")
```
Comment (TRUEJASONFANS) [inline sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java]: /api/room/token/{roomName}

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/util/JwtUtil.java [hunk 4]
```diff
@@ -13,11 +13,11 @@
...
+    private static long ttl = 1000 * 60 * 60 * 24;   //过期时间 1day
```
Comment (TRUEJASONFANS) [inline sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/util/JwtUtil.java]: good job~

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/util/JwtUtil.java (hunk 2 above)
Comment (naijgnay) [reply sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/util/JwtUtil.java]: Ok i will change the key

File: sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java (hunk 3 above)
Comment (naijgnay) [reply sprint-poker-springboot/web/src/main/java/org/zhxie/sprintpoker/controller/RoomController.java]: Ok

**Pull request merged**